build.gradle

```diff
@@ -95,29 +95,38 @@ subprojects {
   dependencies {
     compile "org.apache.logging.log4j:log4j-core:$log4jVersion"
     runtime "org.apache.logging.log4j:log4j-slf4j-impl:$log4jVersion"
-    testCompile project(path: ':', configuration: 'testArchives') 
+    testCompile project(path: ':', configuration: 'testArchives')
   }
-  
+
   task cleanServer << {
     delete 'locator'
     delete 'server1'
     delete 'server2'
   }
   clean.finalizedBy cleanServer
 
+  task replaceVersion(type:Copy) {
+    from "$projectDir/scripts/start.gfsh"
+    from "$projectDir/scripts/stop.gfsh"
+    into (project.getBuildDir())
+    expand(version: "${version}")
+  }
+  compileJava.dependsOn replaceVersion
+
   def geodePath = "${System.env.PATH}${System.getProperty('path.separator')}${installDir}/bin"
   task start(type: Exec, dependsOn: [installGeode, build, cleanServer]) {
     workingDir projectDir
     environment 'GEODE_HOME', installDir
     environment 'PATH', geodePath
-    commandLine 'sh', '-c', "gfsh run --file=${projectDir}/scripts/start.gfsh"
+    commandLine 'sh', '-c', "gfsh run --file=${projectDir}/build/start.gfsh"
```

Comment:
Same as above, wondering why execute from `build`...

Reply:
Please see the previous comment.